eth/catalyst/simulated_beacon.go

```diff
@@ -131,8 +144,7 @@ func (c *SimulatedBeacon) Stop() error {
 
 // sealBlock initiates payload building for a new block and creates a new block
 // with the completed payload.
-func (c *SimulatedBeacon) sealBlock(withdrawals []*types.Withdrawal) error {
-	tstamp := uint64(time.Now().Unix())
+func (c *SimulatedBeacon) sealBlock(withdrawals []*types.Withdrawal, tstamp uint64) error {
```

Comment:
```suggestion
func (c *SimulatedBeacon) sealBlock(withdrawals []*types.Withdrawal, timestamp uint64) error {
```
I know you're just following what was there, but I think `timestamp` is much better param name.